### ISSUE TITLE
allow creating new instances

### ISSUE THREAD

**Issue by pi0** [nuxt/http#3]
Advanced use case. Inside a plugin create a new instance for multi baseURL or custom headers support.

`plugins/http.js`

```js
export default function ({ $http }, inject) {
  inject('blog', $http.create({
    baseURL: 'http://blog.mywebsite.com/api'
  }))
}
```

<div align="right"><sub><em>This question is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/http/issues/c1">#c1</a>)</em></sub></div>

**Comment by warheartvik** [comment]
Hello,

This is my current api set up using axios.

![image](https://user-images.githubusercontent.com/43218752/58195316-47e60280-7c8d-11e9-9db2-d227fdb4de3b.png)

I have a centralized method, which spects the HTTP method, URL, Params and the data:

async Execute (method: any, resource: any, params: any, data: any)

So all my api calls, just send de needed data, like this:

Ejecutar (data: any, direccion: any) {
    return this.Execute('post', direccion,'', data);
}

Is there a way to acomplish this by now using @nuxt/http?

**Comment by pi0** [comment]
Feature implemented by in [v0.5.0](https://github.com/nuxt/http/releases/tag/v0.5.0)

**Comment by pi0** [comment]
@warheartvik You have to create a Nuxt plugin. Please see docs: https://http.nuxtjs.org/guide/advanced.html#header-helpers
